tests/src/Functional/ParagraphsTest.php

```diff
@@ -151,6 +157,49 @@ public function testSocialMediaFollowParagraph(): void {
     $this->assertSession()->pageTextContains('More channels');
   }
 
+  /**
+   * Test paragraphs are wrapped with a spacing.
+   */
+  public function testSpacingBetweenParagraphs(): void {
```

Comment:
rather then doing this we could just add the div count to testLinksBlockParagraph and testSocialMediaFollowParagraph